new/components/basic/form/BasicForm.tsx

```diff
@@ -4,25 +4,28 @@ import BaseButton from "@/components/basic/buttons/BaseButton";
 import BasicTextInput from "./BasicTextInput";
 import ErrorCode from "./ErrorCode";
 import { validationsMap } from "@/lib/validations";
+import BaseContainer from "../layout/BaseContainer";
+import BaseRow from "../layout/BaseRow";
 
 interface BasicFormProps extends PropsWithChildren<FormHTMLAttributes<HTMLFormElement>> {
     onSubmitCallback?: (e: BaseSyntheticEvent, formData: FormData) => Promise<string[]>,
     submitBtnText?: string,
+    styleClass?: string,
 }
 
 export const BasicFormContext = createContext({
-    isSubmitting : false,
-    formData : new FormData()
+    isSubmitting: false,
+    formData: new FormData()
 });
 
-export default function BasicForm({ children, ...props }: BasicFormProps) {
+export default function BasicForm({ children, styleClass, ...props }: BasicFormProps) {
     return (
         <View>
             <BasicFormContext.Provider value={{
-                isSubmitting : false,
-                formData : new FormData()
+                isSubmitting: false,
+                formData: new FormData()
             }}>
-                <BasicFormComponent {...props}>
+                <BasicFormComponent styleClass={styleClass} {...props}>
```

Comment:
why need style for BasicFormComponent? 
Think styling can be done by the children themself.

Reply:
Fixed but can you help fix the container so that when erroe code pop up, the stuff wont move? For now when the error pops up, the input fields items will be pushed upwards